packages/zod-openapi/test/index.test.ts

```diff
@@ -1923,3 +1923,48 @@ describe('Generate YAML', () => {
     expect(() => stringify(doc)).to.not.throw()
   })
 })
+
+describe('Hide Routes', () => {
+  it('Should hide the route', async () => {
```

Comment:
Can you add a test to test `/books` returns a 200 response correctly?

Reply:
Addressed both comments